### DISCUSSION TITLE
注意：大家使用本仓库时请务必遵守许可证

### DISCUSSION THREAD

**Discussion by fumiama** [FloatTech/ZeroBot-Plugin#190] [Announcements]
反面典型：

![73B0E5A38654CD78B2A65DD50B750D11](https://user-images.githubusercontent.com/41315874/163662758-4e05d5c9-4a1d-41a9-a36f-8e4ae00fa6b4.png)

![屏幕快照 2022-04-16 下午1 52 51](https://user-images.githubusercontent.com/41315874/163663592-573f1ad6-539e-463b-b9d6-0824cae20d3c.png)

![屏幕快照 2022-04-16 下午1 41 37](https://user-images.githubusercontent.com/41315874/163663255-32272017-8e96-4b3d-92f5-cd6e8db47f3b.png)

![61A2654DE2E2F48E3B33C7751A3A8EBC](https://user-images.githubusercontent.com/41315874/163663164-d9b8de60-b822-4ce8-a326-b790753a027a.png)

![4C19098C35BD6CDE03F76688BC045B55](https://user-images.githubusercontent.com/41315874/163662796-8d9c99a8-ddf6-4d07-8102-3054ea34da44.jpg)

**Comment by guohuiyuan** [comment]
讲个笑话，开源
